ocs_ci/utility/utils.py

```diff
@@ -1139,7 +1170,8 @@ def email_reports(session):
     msg["Subject"] = (
         f"ocs-ci results for {get_testrun_name()} "
         f"({build_str}"
-        f"RUN ID: {config.RUN['run_id']})"
+        f"RUN ID: {config.RUN['run_id']}) "
+        f"Passed: {percentage_passed:.0f}%"
```

Comment:
I would rather not change the subject of the email. This is basically your API, and as such it should be kept stable.

Reply:
I think percentage pass is great indicator without opening email for high level overview, what will break with addition of pass rate?

Reply:
I'm processing our email archive automatically, so I'm not a fan of changes which provides no additional value and requires changes in my script.

Reply:
I am not sure why you want to block this one for minor addition here, we also have junit for script parsing and the subject was meant mostly for user consumption. we can discuss this in our call again to get +1 , -1 on this.

@dahorak if you want to comment this line for now I think we can take this up next week, instead of holding the PR for long.

Reply:
Commented out in ad6fb10.